Remove extra / character at the beginning of the context

fmartinou opened the pull request:
Fixes #730

ryanjbaxter reviewed (changes requested):
Can you provide a unit test for this?

fmartinou commented:
> Can you provide a unit test for this?

- I added a trivial UT to validate that the context is rewritten as expected (removing first slash...)
- I updated existing UTs so test data contains a context with the extra slash
